### ISSUE TITLE
Chromedriver installation broken

### ISSUE THREAD

**Issue by phansch** [phansch/webdriver-install#60]
```
called `Result::unwrap()` on an `Err` value: No support for unarchiving %3C, yet
```
It looks the the way to fetch chromedriver versions has changed

**Comment by sayav** [comment]
Yeah it looks like for versions >115, the URL for getting the latest ChromeDriver release number has changed

https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE_{build_version}

**Comment by mikebirdgeneau** [comment]
The versions / downloads look to be available from this JSON endpoint:
[https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json](https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json)

**Comment by mikebirdgeneau** [comment]
This commit on my fork might need some tweaks (still learning Rust), but seems to work with the latest Chromedriver:
https://github.com/phansch/webdriver-install/commit/bbefab85b588f838317e4bdac7a3c94d9d43436f
